### ISSUE TITLE
Not working on Android API 24

### ISSUE THREAD

**Issue by brunolemos** [BranchMetrics/react-native-branch-deep-linking#277]
Hi,

It's working fine on iOS and Android API 22,
but on Android API 24 I get this error after calling `branch.subscribe`: 

`Trouble initializing Branch.  The request was invalid.`

I followed the installation instructions in the readme (manifest, .gradle files, .java files, api key, ...) but still having this error.
Any idea?

Thanks.

--
react native 0.48
react-native-branch 2.1.0
macOS 10.12.6
Android Emulator API 24

**Comment by brunolemos** [comment]
I found that the problem occurs on Android API 24 but not on API 22.

**Comment by jdee** [comment]
A support ticket has been opened for this. Closing.

**Comment by brunolemos** [comment]
For future reference: The support asked me to test their example https://github.com/BranchMetrics/react-native-branch-deep-linking/tree/master/examples/webview_example and it worked fine here on API 24. So just my app seems to be having issue.

But I tested my app on API 22 and API 26 and both worked fine so I decided to ignore this issue for now using a try catch around `branch.subscribe` method as the sharing feature is not that crucial to the app at the moment.
